olp-cpp-sdk-dataservice-read/tests/CatalogClientTest.cpp

```diff
@@ -1634,10 +1634,14 @@ TEST_P(CatalogClientMockTest, DISABLED_GetPartitionsCancelLayerVersions) {
       waitForCancel, pauseForCancel, {200, HTTP_RESPONSE_LAYER_VERSIONS});
```

Comment:
Dont you need to change {200, HTTP_RESPONSE_LAYER_VERSIONS} to send a lookup response back instead of HTTP_RESPONSE_LAYER_VERSIONS

Reply:
Good point, will add that for consistency. Although that has no effect since the test cancels the request right at the time of response is generated.